models/options/OptionSet.py

```diff
@@ -72,7 +72,7 @@ def __init__(self):
         self.mystery_settings = MysteryOptionSet()
 
         # Starting setup
-        self.starting_level = get_option_default_value("StartingLevel")
+        self.starting_level = self.random_starting_level = get_option_default_value("StartingLevel")
```

Comment:
This might not work as expected, please verify.
`get_option_default_value()` reads data from the sqlite db in the root dir, and that db gets built from the base rom data.
This default value is used if the associated key is not provided within the settings yaml.
The base rom sets StartingLevel to 1, so this might mess with random starting levels if no StartingLevel keyvalue pair is provided.